contracts/claims/data/crab/data/nft_events.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Darwinia & Crab NFT",
+  "timestamp": "2022-08-16T00:00:00.000Z",
+  "version": { "major": 0, "minor": 1, "patch": 0 },
+  "logoURI": "https://portal.evolution.land/logos/darwinia.png",
+  "events": [
+    {
+      "chainId": 44,
+      "address": "0xB29DA7C1b1514AB342afbE6AB915252Ad3f87E4d",
```

Comment:
0xB29DA7C1b1514AB342afbE6AB915252Ad3f87E4d => 0xA561c8F6AC9eCb31e0793fBe2CCEc136a7e4bE84

The address is claims contracts ? @WoeOm

Reply:
Should be the claims contract address